submissions/mayowaShoyinka/htmlForms/product.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
You are meant to style the product page in the html folder not create a new product page in html-forms. @MayowaShoyinka